### ISSUE TITLE
{Feauture Request] Provide yum/dnf repo with RPMs

### ISSUE THREAD

**Issue by pschiffe** [TheGoddessInari/rambox#33]
Hi @TheGoddessInari , thanks for all the great work you put into this fork. I have a suggestion here - provide yum repository for RPM based Linux distributions for easier installing and updating of the Rambox RPM package. I don't know whether the appveyor has such feature, but for example [Atom Editor](https://flight-manual.atom.io/getting-started/sections/installing-atom/#red-hat-and-centos-yum-or-fedora-dnf) is using [packagecloud](https://packagecloud.io/). Other options could be [copr.fedorainfracloud.org](https://copr.fedorainfracloud.org/) or [build.opensuse.org](https://build.opensuse.org/).

Let me know if you need help with this. After you release 0.5.18 version, I'll switch to this fork instead of Rambox CE.

**Comment by TheGoddessInari** [comment]
Ah, I meant to reply to this soon after you posted it. I looked into FedoraInfraCloud and OpenSuSE build stuff, but could probably use help if it's (relatively) as straightforward as it sounds. It sounds like OBS has or had some integration with that, and I spent a few days trying to figure it out, but it was confusing. I'll look into it some over the weekend, maybe I'll grok it this time.

I could investigate packagecloud after the official renaming (which'll be just after 0.5.18 I think).

**Comment by pschiffe** [comment]
Yeah, I think that packagecloud has most features from that three options.

**Comment by flocke** [comment]
Any plans for this? I would also love to have a package repository for this project, specifically for DEB packages. From a first look the OBS supports building multiple package formats, so you would be able to provide RPM, DEB, ... packages from one source.
But maybe this should be only done after the rebranding so we have one consistent package name.
